Defs/Ammo/HighCaliber/30x170mm.xml

```diff
@@ -0,0 +1,301 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+
+	<ThingCategoryDef>
+		<defName>Ammo30x170mm</defName>
+		<label>30x170mm</label>
+		<parent>AmmoHighCaliber</parent>
+		<iconPath>UI/Icons/ThingCategories/CaliberHighCaliber</iconPath>
+	</ThingCategoryDef>
+
+	<!-- ==================== AmmoSet ========================== -->
+
+	<CombatExtended.AmmoSetDef>
+		<defName>AmmoSet_30x170mm</defName>
+		<label>30x170mm</label>
+		<ammoTypes>
+			<Ammo_30x170mm_AP>Bullet_30x170mm_AP</Ammo_30x170mm_AP>
+			<Ammo_30x170mm_Incendiary>Bullet_30x170mm_Incendiary</Ammo_30x170mm_Incendiary>
+			<Ammo_30x170mm_HE>Bullet_30x170mm_HE</Ammo_30x170mm_HE>
+			<Ammo_30x170mm_Sabot>Bullet_30x170mm_Sabot</Ammo_30x170mm_Sabot>
+		</ammoTypes>
+		<similarTo>AmmoSet_Autocannon</similarTo>
+	</CombatExtended.AmmoSetDef>
+
+	<!-- ==================== Ammo ========================== -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" Name="Ammo30x170mmBase" ParentName="MediumAmmoBase" Abstract="True">
+		<description>Large caliber cartridge used by autocannons.</description>
+		<statBases>
+			<Mass>0.71</Mass>
```

Comment:
[Source](https://ordtech-industries.com/wp-content/uploads/2023/09/OMI-30X170mm.pdf) puts the overall mass at 810g. Currently, the casing mass is a fair bit under other projectiles listed on the sheet.

Dimensions entered into the sheet look fine. [Reference.](https://old.municion.org/30mm/30x170Rarden.htm)

Reply:
Correct, it's probably a typo being carried over from the original file/PR.